src/components/DashboardLayout/NotificationPopover.tsx

```diff
@@ -0,0 +1,36 @@
+import {
+  faBell,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Popover } from '@headlessui/react';
+import { NotificationContext } from 'data/NotificationContext';
+
+import NotificationPanel from './NotificationPanel';
+import { useContext, useEffect, useState } from 'react';
+import clsx from 'clsx';
+
+export const NotificationPopover = () => {
+  const { notifications, ongoingNotifications } = useContext(NotificationContext);
+  const [ newNotifications, setNewNotifications ] = useState<boolean>(false);
+
+  useEffect(() => {
+    setNewNotifications(notifications.length > 0 || ongoingNotifications.length > 0);
+  }, [notifications, ongoingNotifications]);
+    
+  return (
+    <Popover className="relative">
+      <Popover.Button
+        as={FontAwesomeIcon}
+        icon={faBell}
+        className={clsx(
+          "w-4 select-none hover:cursor-pointer ui-open:text-gray-300",
+          newNotifications ? "ui-not-open:text-white/75 ui-not-open:hover:text-white/90" :
```

Comment:
The brighter icon isn't very noticeable.
I think it might be better if we add a simple CSS animation that makes it flash?
or just a onetime animation when it goes from the "no new notification" state to the "new notification" state.
or add a red dot?
whichever one looks better

Reply:
added a red dot

Reply:
nice